lib/modules/dosomething/dosomething_reportback/includes/Reportback.php

```diff
@@ -121,12 +121,22 @@ public static function find(array $filters = []) {
       throw new Exception('No reportback data found.');
     }
 
-    foreach($results as $item) {
-      // @TODO: remove need for passing variable for constructor check.
-      $reportback = new static(['ignore' => TRUE]);
-      $reportback->build($item);
+    if (isset($filters['load_user_data'])) {
```

Comment:
oh this is so smart, thanks Diego! I was trying to figure out how to extract the code so I wouldn't repeat lines - so helpful!